### PR TITLE
@xtina-starr - render the homepage regardless, even if we fail to fetch the rails

### PR DESCRIPTION
**Pull request by broskoski**
This is more of a stopgap solution. This "cheats" the metaphysics promise by always resolving no matter what. 

Obviously we should think of a better way to move forward with the partial errors, maybe we can use this PR to discuss alternatives.

cc @craigspaeth @alloy

### CODE REVIEW

File: apps/home/test/routes.coffee [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      @req.query['redirect_uri'] = '/awesome-fair'
+      @req.user = {}
+      routes.redirectLoggedInHome @req, @res
+      @res.redirect.args[0][0].should.equal '/awesome-fair'
+
+    it 'redirects logged in users to redirect location', ->
+      @req.body['redirect-to'] = '/awesome-fair'
+      @req.user = {}
+      routes.redirectLoggedInHome @req, @res
+      @res.redirect.args[0][0].should.equal '/awesome-fair'
```
Comment (alloy) [inline apps/home/test/routes.coffee]: Might want to update the description of these two tests slightly to indicate where the difference lies, took me a few sec to figure out.

File: apps/home/routes.coffee [hunk 2]
```diff
@@ -25,6 +25,14 @@ positionWelcomeHeroMethod = (req, res) ->
   res.cookie 'hide-welcome-hero', '1', expires: new Date(Date.now() + 31536000000)
   method
 
+maybeFetchHomepageRails = (req)->
+  deferred = Q.defer()
+  metaphysics(query: query, req: req)
+    .then (data) -> deferred.resolve data
+    .catch -> deferred.resolve home_page: { artwork_modules: [] }
```
Comment (alloy) [inline apps/home/routes.coffee]: So to be clear, the `metaphysics` function in Force rejects a promise when the GraphQL JSON response contains an `errors` entry? Or is this about connection failures between Force and metaphysics?

**Comment by broskoski** [comment]
💚

File: apps/home/routes.coffee (hunk 2 above)
Comment (alloy) [inline apps/home/routes.coffee]: :+1:

**Pull request merged**